debug_poisson_group.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
```

Comment:
can you come up with a unit test (I don't see an easy one right now) to get more rigorous testing ?